__test__/Block.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
```

Comment:
Do we actually need the Adapter imports?

Reply:
They are necessary, but could probably be handled in a setup file in the `block-scripts` package, instead of directly imported into the block tests.

Reply:
Are you sure? I stopped using them a long time ago, and they're not imported in our enzymeBlock helper.

Reply:
In zero-Blocks, they're handled in the jest setup file: https://github.com/Volusion2Dev/zero-Blocks/blob/master/jest.setup.js

Reply:
Got it. Why not replicate that?

Reply:
@cameron-jones-volusion What's the status on this? Leaving as is or moving to a `jest.setup.js` file?

Reply:
I'd like to move it to a setup file in `block-scripts` in a later PR, mostly due to the increased difficulty of testing the change across two packages at once. Isolating the change in it's own PR should make it easier to test it effectively.